test_data/std/poseidon_gl_test.asm

```diff
@@ -55,7 +55,7 @@ machine Main {
         assert_eq C, 14940024623104903507;
         assert_eq D, 10772674582659927682;
 
-        A, B, C, D <== poseidon(18446744069414584321, 18446744069414584321, 18446744069414584321, 18446744069414584321, 18446744069414584321, 18446744069414584321, 18446744069414584321, 18446744069414584321, 0, 0, 0, 0);
+        A, B, C, D <== poseidon(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
```

Comment:
Does it panic otherwise?

Reply:
These test vectors are the same as polygon's. I guess they were chosen exactly to test that the result is the same as if you change the inputs to 0s. The last 4 are 0s because they're the `cap` that are used to "change" the hash function

Reply:
Exactly, they come from the JSON linked in the comment a few lines above.